In [1]:
#DEPENDENCIES & INITIALISATIONS
from IPython.display import HTML #For embedding videos
import math #Basic math functionality
import numpy as np #Numpy for working with arrays
import matplotlib.pyplot as plt #Plotting functionality 
#HTML('<center><iframe width="560" height="315" src="#" frameborder="0" allowfullscreen></iframe></center>')
print("- Dependencies imported 👍")
print("- Remember to select 'Kernel-> Restart & Run All' on first load to show embedded Youtube videos 🎬")
print("- Author: Dr SPCarroll, Sept 2020, Rev 0")

- Dependencies imported 👍
- Remember to select 'Kernel-> Restart & Run All' on first load to show embedded Youtube videos 🎬
- Author: Dr SPCarroll, Sept 2020, Rev 0


# Week 9: Momentum-based Analysis Methods
Reference: Chapter 16 of Engineering Mechanics - Statics and Dynamics Principles, Bedford A. & Fowler, W., 2003

## Question 1

Consider a motorcycle starting from rest at $t=0$ seconds and travelling on a circular track with radius $R=400\:m$. The combined weight of the bike and rider is $150\:kg$ The tangential component of the total force on the motorcycle between $0\leq t \leq 30$ is,

\begin{equation*}
\sum F_t = 200-6t\:N
\end{equation*}

- (a) What is the magnitude of the velocity at $t=30$ seconds
- (b) What is the average of the tangential component of the total force from $t=0$ to $t=30$ seconds.
- (c) What is the average of the normal component of the total force from $t=0$ to $t=30$ seconds.

**Solution 1(a)**

Since we are considering the motion along a curved path, we can use the scalar form of the principle of impulse and momentum,

\begin{equation*}
\int_{t_1}^{t_2} \sum F_t\:\mathrm{d}t = m v_2 - m v_1
\end{equation*}

\begin{equation*}
\int_{0}^{30} (200-6t)\:\mathrm{d}t = 150v_2
\end{equation*}

Integrating between the limits and solving for $v_2$,

\begin{equation*}
v_2 = 22\:m/s
\end{equation*}

**Solution 1(b)**

We can use equation (6) to solve for the average force,

\begin{equation*}
(t_2-t_1)\sum F_{t,avg} = m v_2 - m v_1
\end{equation*}

\begin{equation*}
(30-0)\sum F_{t,avg} = (150)(22) 
\end{equation*}

\begin{equation*}
F_{t,avg} = 110\:N
\end{equation*}

**Solution 1(c)**

We need to first determine an expression for the normal component of force. From part (a) above we can obtain an expression for the velocity as a function of time,

\begin{equation*}
v = \frac{200t - 3t^2}{150}
\end{equation*}

We're dealing with circular motion and we have previously seen that the acceleration in the normal direction can be obtained as a function of the velocity,

\begin{equation*}
a_n = \frac{v^2}{R}
\end{equation*}

Now considering Newton's second law in the normal direction we have,

\begin{equation*}
\sum F_n = m\frac{v^2}{R}
\end{equation*}

Noting that $R=400\:m$ and $m=150\:kg$, we obtain our expression for the normal force as a function of time,

\begin{equation*}
\sum F_n = \frac{(200t-3t^2)^2(150)}{(150^2)(400)}
\end{equation*}

Now, using the scalar form of equation (5) we have,

\begin{equation*}
\sum F_{n,avg} = \frac{1}{30-0}\int_{0}^{30} \frac{(200t-3t^2)^2(150)}{(150^2)(400)}\:\mathrm{d}t
\end{equation*}

Using SymPy to complete the integration,

In [2]:
import sympy as sym #Import SymPy
t = sym.Symbol('t') #Define symbol for t 
f = (150*(200*t - 3*t**2)**2)/(400*150**2) #Construct the function to integrate
Fn = (1/30)*sym.integrate(f, (t, 0, 30)) #Evaluate integral between the limits
print('1(c). The average of the normal component of force is {one} N'.format(one=round(Fn,1)))

1(c). The average of the normal component of force is 89.3 N


## Question 2

A golfer striking golf balls in the lab is recorded by a high-speed camera that records an image every $0.001$ seconds. The golf balls have a diameter of $43\:mm$ and weight $50\:g$ (grammes). The club is in contact with the ball for $0.0006$ seconds. If the ball travels $49\:mm$ between images on a linear trajectory orientated at $21^\circ$ counterclockwise to the positive x-axis, determine the average value of the impulse force exerted on the ball during contact with the club, $\sum \mathbf{F}_{avg}$.

![title](img/T2W9/img2.jpg)
<center>Question 2.</center>

**Solution 2**

We can obtain the average force over the duration using equation (6),

\begin{equation*}
(t_2-t_1) \sum \mathbf{F}_{avg} = m\mathbf{v}_2 - m\mathbf{v}_1
\end{equation*}

First we work out the magnitude of the ball's velocity in $m/s$.

In [3]:
d = 0.049 #(m) Distance travelled in 0.001s
vMag = d*1000 #(m/s) Distance travelled in 1 second (velocity)
print('The velocity magnitude of the ball is {one} m/s'.format(one=round(vMag,1)))

The velocity magnitude of the ball is 49.0 m/s


Next we need to determine the $x$ and $y$ component of the velocity in order to determine the velocity vector $\mathbf{v}_2$.

In [4]:
theta_deg = 21 #(degrees)
theta = theta_deg*(math.pi/180)
vx = vMag*math.cos(theta) #(m/s) Horizontal component of velocity
vy = vMag*math.sin(theta) #(m/s) Vertical component of velocity
print("The horizontal component of the ball's velocity is {one} m/s".format(one=round(vx,2)))
print("The vertical component of the ball's velocity is {one} m/s".format(one=round(vy,2)))

The horizontal component of the ball's velocity is 45.75 m/s
The vertical component of the ball's velocity is 17.56 m/s


Therefore the velocity vector is $\mathbf{v}_2 = 45.75\:\mathbf{i} + 17.56\:\mathbf{j}$. Noting that the ball starts from rest, $\mathbf{v}_1=0$, we can now solve for $\sum \mathbf{F}_{avg}$.

In [5]:
delT = 0.0006 #(s) Contact time t2-t1
m = 50/1000 #(kg)
Fx = (m/delT)*vx
Fy = (m/delT)*vy
print("The horizontal component of force is {one} N".format(one=round(Fx,1)))
print("The vertical component of force is {one} N".format(one=round(Fy,1)))

The horizontal component of force is 3812.1 N
The vertical component of force is 1463.3 N


Therefore the average force vector is,

\begin{equation*}
\sum \mathbf{F}_{avg} = (3812\:\mathbf{i} + 1463\:\mathbf{j})\:N
\end{equation*}

## Question 3

The Apollo command service module (CSM) attempts to dock with the Soyuz capsule in July 15, 1975. Their masses are $m_A = 18000 \:kg$ (CSM) and $m_B = 6600 \:kg$ (Soyuz). The Soyuz can be considered stationary in this analysis. The CSM approaches with velocity

\begin{equation*}
v_A = 0.2\mathbf{i} + 0.03\mathbf{j} - 0.02\mathbf{k}\:m/s
\end{equation*}

- (a) If the first attempt at docking is successful, what is the velocity of the center of mass of the combined vehicles afterward?

- (b) If the first attempt is unsuccessful and the coefficient of restitution of the resulting impact is $e = 0.95$, what are the velocities of the two spacecraft after the impact? Assume that a direct central impact occurs along the x-direction.

![title](img/T2W9/img1.jpg)
<center>Question 3.</center>

**Solution 3(a)**

Since both masses can be considered as a single mass after the impact, we only have one unknown to solve for and can use equation (13) to solve for the velocity of the combined mass.

\begin{equation*}
\mathbf{v} = \frac{m_A\mathbf{v}_A + m_B\mathbf{v}_B}{(m_A+m_B)}
\end{equation*}

In [6]:
mA = 18000 #(kg)
mB = 6600 #(kg)
vAi = 0.2 #(m/s) x-component of velocity of mass A
vAj = 0.03 #(m/s) y-component of velocity of mass A
vAk = -0.02 #(m/s) z-component of velocity of mass A

vx = (mA/(mA+mB))*vAi #(m/s) x-component of combined velocity post-docking
vy = (mA/(mA+mB))*vAj #(m/s) y-component of combined velocity post-docking
vz = (mA/(mA+mB))*vAk #(m/s) z-component of combined velocity post-docking

print("The x-component of the velocity of the combined mass is {one} m/s".format(one=round(vx,3)))
print("The y-component of the velocity of the combined mass is {one} m/s".format(one=round(vy,3)))
print("The z-component of the velocity of the combined mass is {one} m/s".format(one=round(vz,3)))

The x-component of the velocity of the combined mass is 0.146 m/s
The y-component of the velocity of the combined mass is 0.022 m/s
The z-component of the velocity of the combined mass is -0.015 m/s


\begin{equation*}
\mathbf{v} = (0.146\:\mathbf{i} + 0.022\:\mathbf{j}-0.015\:\mathbf{k})\:m/s
\end{equation*}

**Solution 3(b)**

Now we have two post-impact velocities to solve for and so we need two equations; the first will be the conservation of linear momentum equation in the $x$ direction,

\begin{equation*}
m_A\mathbf{v}_{A,x} + m_B\mathbf{v}_{B,x} = m_A\mathbf{v}'_{A,x} + m_B\mathbf{v}'_{B,x}
\end{equation*}

Noting the velocity of the Soyuz is zero before impact, $\mathbf{v}_{B,x}=0$,

\begin{equation*}
(18000)(0.2) + (6600)(0) = (18000)(\mathbf{v}'_{A,x}) + (6600)(\mathbf{v}'_{B,x})
\end{equation*}

\begin{equation*}
18000\mathbf{v}'_{A,x} + 6600\mathbf{v}'_{B,x} = 3600
\end{equation*}



To obtain our second equation, we use equation (23) for the coefficient of restitution, 

\begin{equation*}
e = \frac{\mathbf{v}'_{B,x} - \mathbf{v}'_{A,x}}{\mathbf{v}_{A,x}-\mathbf{v}_{B,x}}
\end{equation*}

\begin{equation*}
0.95 = \frac{\mathbf{v}'_{B,x} - \mathbf{v}'_{A,x}}{0.2-0}
\end{equation*}

\begin{equation*}
-\mathbf{v}'_{A,x} + \mathbf{v}'_{B,x} = 0.19
\end{equation*}

Solving both equations simultaneously yields,

\begin{equation*}
\mathbf{v}'_{A} = (0.095\mathbf{i} + 0.03\mathbf{j}-0.02\mathbf{k})\:m/s
\end{equation*}

\begin{equation*}
\mathbf{v}'_{B} = 0.285\mathbf{i}\:m/s
\end{equation*}

Note that only the $x$ or $\mathbf{i}$ component of the velocity of each mass had been altered by the collision. 

## Question 4

Suppose that you investigate an accident in which a $1300\:kg$ car $A$ struck a parked $1200 \:kg$ car $B$. All four of car $B$'s wheels were locked and skid marks indicate that it slid $8\:m$ after the impact. If you estimate the coefficient of kinetic friction between $B$'s tires and the road to be $\mu_k = 0.8$ and the coefficient of restitution of the impact to be $e = 0.4$, what was $A$'s velocity $v_A$ just before the impact?

**Hint:** Start by finding the velocity of car B immediately after the impact.


![title](img/T2W9/img3.jpg)
<center>Question 4.</center>

**Solution 4**

We know how far car $B$ has moved after the impact so we can start by working back to determine its velocity immediately after the impact. Once we know the velocity of car $B$ after the impact we can use conservation of momentum to determine the velocity of car $A$ before the impact.

Considering car $B$ post impact; the work done by the friction force between the wheels and road is,

\begin{equation*}
U = m_B\:g\:\mu_k\:d
\end{equation*}

where $d$ is the distance the car has skidded. We know this is equal to the change in kinetic energy and since the final velocity is zero, this is given by,

\begin{equation*}
U = \frac{1}{2} m_B v_B^2
\end{equation*}

where $v_B$ is the post-impact velocity we are trying to find. 

In [7]:
uk = 0.8 # Coefficient of friction
d = 8 #(m) Skid distance
vb = math.sqrt(2*9.81*uk*d) #(m/s) 
print("The post-impact velocity of car B is {one} m/s.".format(one=round(vb,2)))

The post-impact velocity of car B is 11.21 m/s.


We can now use conservation of momentum and the coefficient of restitution to determine $v_A$. Considering conservation of momentum first,

\begin{equation*}
m_A\:v_A + m_B\:v_B = m_A\:v_A' + m_B\:v_B'
\end{equation*}

Since car $B$ is stationary before the impact,

\begin{equation*}
(1300)\:v_A = (1300)\:v_A' + (1200)\:(11.21)
\end{equation*}

\begin{equation*}
\boxed{
1300\:v_A -1300\:v_A' =13452
}
\end{equation*}

Now using the coefficient of restitution,

\begin{equation*}
v_A\:e = v_B'-v_A'
\end{equation*}

\begin{equation*}
\boxed{
0.4\:v_A + v_A' = 11.21
}
\end{equation*}

Solving the preceding two boxed equations yields $v_A'=5.05\:m/s$ and $v_A = 15.4\:m/s$ which is the velocity of car $A$ just before impact. 

## Question 5

A ball bouncing on the ground achieves two successive bounce heights of $h_1=5.1\:cm$ and $h_2 = 3.1\:m$. Estimate the coefficient of restitution.

![title](img/T2W9/img4.jpg)
<center>Question 5.</center>

**Solution 5**

We start by considering the sum of the kinetic and potential energy for the ball when it is at height $h_1$ and its velocity magnitude is zero,

\begin{equation*}
\frac{1}{2}m(0^2) + mgh_1 
\end{equation*}

This must equal the sum of total energy when the ball has fallen, just before impact when it's height is zero and it has velocity $v_{\text{ball}}$,

\begin{equation*}
\frac{1}{2}mv_{\text{ball}}^2 + mg(0)
\end{equation*}

Equating the energy statements,

\begin{equation*}
v_{\text{ball}} = \sqrt{2gh_1}
\end{equation*}

We can follow the same procedure for the post impact flight of the ball to a height of $h_2$ to yield an expression for the post impact velocity,

\begin{equation*}
v_{\text{ball}}' = \sqrt{2gh_2}
\end{equation*}

We can now consider our expression for the coefficient of restitution noting that the ground is stationary $(v_B = v_B'=0)$,

\begin{equation*}
e=\frac{-v_A'}{v_A} = \sqrt{\frac{h_2}{h_1}}
\end{equation*}

In [8]:
h1 = 5.1
h2 = 3.1
e=math.sqrt(h2/h1)
print('The coefficient of restitution is {one}'.format(one=round(e,2)))

The coefficient of restitution is 0.78


## Answers

1. $22\:m/s$, $110\:N$, $89.3 \:N$
2. $\sum \mathbf{F}_{avg} = (3812\:\mathbf{i} + 1463\:\mathbf{j})\:N$
3.  - $\mathbf{v} = (0.146\:\mathbf{i} + 0.022\:\mathbf{j}-0.015\:\mathbf{k})\:m/s$, 
    - $\mathbf{v}'_{A} = (0.095\mathbf{i} + 0.03\mathbf{j}-0.02\mathbf{k})\:m/s$
    - $\mathbf{v}'_{B} = 0.285\mathbf{i}\:m/s$
4. $15.4\:m/s$
5. $0.78$